### PR TITLE
Remove CoreData from project

### PR DESCRIPTION
**Pull request by Qeezii**
# Description 📝

Removed CoreData from the project

---

### CODE REVIEW

**Review by Kengii — APPROVED** [review]

**Review by arti1233 — APPROVED** [review]

File: Runar/Runar/Controllers/Generator flow/SelectRune/SelectRuneCell.swift [hunk 1]
```diff
@@ -43,14 +43,11 @@ class SelectRuneCell: UICollectionViewCell {
...
-    public func setRune(_ rune: GeneratorRuneCoreDataModel, _ indexPath: IndexPath) {
+    public func setRune(_ rune: GenerationRuneModel, _ indexPath: IndexPath) {
         self.model = rune
         self.indexPath = indexPath
 
-        guard let imageData = rune.runeImage?.image,
-              let image = UIImage(data: imageData) else { return }
-
-        runeImage.setBackgroundImage(image, for: .normal)
+        runeImage.setBackgroundImage(rune.image.image, for: .normal)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Generator flow/SelectRune/SelectRuneCell.swift]: @Qeezii rune.image.image - has image another image?

File: Runar/Runar/Controllers/Generator flow/SelectionRuneVC.swift [hunk 2]
```diff
@@ -329,8 +322,7 @@ public class SelectionRuneVC: UIViewController, UIGestureRecognizerDelegate {
             for cell in (self.selectedRunesView.visibleCells as! [SelectedRuneCell]).sorted(by: {c1, c2 in return c1.indexPath.row < c2.indexPath.row} ) {
                 if !cell.isSelected {
                     guard let title = rune.model?.title,
-                          let imageData = rune.model?.runeImage?.image,
-                          let image = UIImage(data: imageData),
+                          let image = rune.model?.image.image,
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Generator flow/SelectionRuneVC.swift]: @Qeezii the same for image.image. Think, please, about naming and model structure.

File: Runar/Runar/Services/Memory/DataManager.swift [hunk 3]
```diff
@@ -69,39 +69,26 @@ final class DataManager {
     // MARK: - Load library
     private func loadLibraryData() {
         // Download data from server
-        guard let libraryData = RunarApi.getLibratyData() else { fatalError("Library is empty") }
+        guard let libraryData = RunarApi.getLibratyData() else {
+            // TODO: - Provide for the processing of the case if it was not possible to download data from the server
+            return print("Library is empty")
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Services/Memory/DataManager.swift]: @Qeezii provide, please, correct print/return

guard let libraryData = RunarApi.getLibratyData() else {
             // TODO: - Provide for the processing of the case if it was not possible to download data from the server
             print("Library is empty")
             return
}

File: Runar/Runar/Services/Memory/DataManager.swift [hunk 4]
```diff
@@ -69,39 +69,26 @@ final class DataManager {
...
-        libraryIsLoaded = true
     }
 
     // MARK: - Load generator
     private func loadGeneratorData() {
         // Download data from server
-        guard let runesData = RunarApi.getRunesData() else { fatalError("Runes is empty") }
+        guard let runesData = RunarApi.getRunesData() else {
+            // TODO: - Provide for the processing of the case if it was not possible to download data from the server
+            return print("Runes is empty")
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Services/Memory/DataManager.swift]: @Qeezii the same issue.

**Review by AliaksandrNiehaj — APPROVED** [review]

**Pull request merged**